src/main/java/net/glease/ggfab/mte/MTE_AdvAssLine.java

```diff
@@ -502,6 +522,20 @@ public boolean onRunningTick(ItemStack aStack) {
         return true;
     }
 
+    private ItemStack getInputBusContent(int index) {
+        if (mItemCache[index] == NULL) {
+            ItemStack stuff = null;
+            if (index < mInputBusses.size()) {
+                GT_MetaTileEntity_Hatch_InputBus bus = mInputBusses.get(index);
+                if (isValidMetaTileEntity(bus)) {
```

Comment:
call bus.isValid() instead. Since [this](https://github.com/GTNewHorizons/GT5-Unofficial/pull/2267) gt5u PR.